lib/generators/simple_form/templates/config/initializers/simple_form_bootstrap.rb

```diff
@@ -126,7 +126,7 @@
     b.use :html5
     b.optional :readonly
     b.use :label, class: 'control-label'
-    b.wrapper tag: 'div', class: 'form-inline' do |ba|
+    b.wrapper tag: 'div', class: 'form-inline col-sm-10' do |ba|
```

Comment:
This seems to only be necessary in the horizontal form, right? How would that affect a non horizontal form?

Also, other horizontal forms tend to use `col-sm-3` in the label and `col-sm-9` in the field, any particular difference?

Thanks.